packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -563,6 +563,31 @@ describe('KeyringController', () => {
               },
             );
           });
+
+          it('should remove encryptionKey and encryptionSalt from state when cacheEncryptionKey is false', async () => {
```

Comment:
Nit: Maybe we should only run this if `cacheEncryptionKey` is `false`?

Since these tests are in a block that gets run for each value of `cacheEncryptionKey`, currently this is being run twice redundantly.